code/modules/urist/items/clothes/uristclothes.dm

```diff
@@ -1813,3 +1813,16 @@ Update 26/07/2014 - All generic clothing goes under obj/item/clothing/under/uris
 		bio = ARMOR_BIO_SMALL,
 		rad = ARMOR_RAD_MINOR
 		)
+
+/obj/item/clothing/under/contortionist
+	name = "contortionist's jumpsuit"
+	desc = "A light jumpsuit useful for squeezing through narrow vents."
+	icon_state = "robotics2"
+	item_state = "robotics2"
+
+/obj/item/clothing/under/contortionist/proc/check_clothing(mob/user as mob)
+	//Allowed to wear: glasses, shoes, gloves, pockets, mask, and jumpsuit (obviously)
+	var/list/slot_must_be_empty = list(slot_back,slot_handcuffed,slot_legcuffed,slot_l_hand,slot_r_hand,slot_belt,slot_head,slot_wear_suit)
+	for(var/slot_id in slot_must_be_empty)
+		to_chat(user, SPAN_WARNING("You can't fit inside while wearing those bulky clothes."))
+		return 0
```

Comment:
I don't see how this will ever do anything but return 0 instantly every time. This is not checking anything about the slots AFAICT, it just blanket-returns 0.

Reply:
Separate issue altogether: this unnecessarily mixes logic and UI together. 

Check functions should be pure logic, if they return false/0/null? then we can decide how to communicate that to the user in... whatever we're doing (ventcrawling, in this case). 

Otherwise we if we want to check before committing to doing stuff, or similarly when the check inevitably gets repurposed for something else, we'll get random irrelevant chat message spam.